src/workerd/server/server.c++

```diff
@@ -1764,7 +1764,8 @@ private:
     Connection(HttpListener& parent, kj::Maybe<kj::String> cfBlobJson)
         : parent(parent), cfBlobJson(kj::mv(cfBlobJson)),
           http(parent.timer, parent.headerTable, *this, kj::HttpServerSettings {
-            .errorHandler = *this
+            .errorHandler = *this,
+            .webSocketCompressionMode = kj::HttpServerSettings::MANUAL_COMPRESSION
```

Comment:
Should we also update the `newHttpClient` calls in `ExternalHttpService` and `NetworkService`?

Reply:
https://github.com/cloudflare/workerd/pull/69/commits/79966c974b7948dadc066e5dc86544c074f00349